lib/presentation/bloc/movie_genres_bloc.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:async';
+
+import '../../core/bloc/bloc.dart';
+
+import '../../data/model/data_state.dart';
+
+import '../../domain/usecase/implementation/get_movie_genres_usecase.dart';
+import '../../domain/usecase/usecase_interface.dart';
+
+class MovieGenresBloc implements Bloc {
+  UseCase useCase = GetMovieGenresUseCase();
```

Comment:
This could be added as a parameter of the MoviesGenresBloc constructor like
MovieGenresBloc({
      this.useCase
});

final GetMovieGenresUseCase useCase;

Reply:
done